sdk/eventhub/event-hubs/src/eventHubConsumerClient.ts

```diff
@@ -103,7 +109,7 @@ export class EventHubConsumerClient {
    * This is likely to be similar to <yournamespace>.servicebus.windows.net.
    */
   get fullyQualifiedNamespace(): string {
-    return this._context.config.host;
+    return this._contextManager.getGatewayConnectionContext().config.host;
```

Comment:
Would it be simpler to just have _contextManager export host and entity path directly? Then it conceals one more detail from consumers.